doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -216,6 +216,13 @@ nRF IEEE 802.15.4 radio driver
 Wi-Fi
 -----
 
+* Added:
+
+  * The :kconfig:option:`NRF_WIFI_FW_PATCH_DFU` Kconfig option to enable DFU support for nRF70 Series devices.
+    This allows firmware patches for signed images to be sent to multi-image targets over Wi-Fi.
+
+  * The :ref:`ug_nrf70_fw_patch_update` user guide in the :ref:`ug_nrf70_developing` section that describes how to perform a DFU for firmware patches using Wi-Fi.
```

Comment:
I would suggest to move line 224 under Documentation section after line 1367, as this doc is not added under Protocols section.